add-ons/ansible-service-broker/README.md

```diff
@@ -0,0 +1,70 @@
+Ansible Service Broker Addon
+======================
```

Comment:
This is a suggestion which you can feel free to ignore, but I find that it's easier to both maintain and understand headings which use `#`.